source/_integrations/binary_sensor.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how-to setup binary sensors with Home Assistant.
 ha_category:
   - Binary Sensor
 ha_release: 0.9
+ha_iot_class: Calculated
```

Comment:
There is no IoT class for this integration:

```suggestion
ha_iot_class: ~
```

Reasoning: This integration cannot be used by itself, and its uses come from implementation on top of this one.

Setting it to `~` makes it explicit null/none.